R/utilities.R

```diff
@@ -884,7 +884,7 @@ as_continuous_pal <- function(x, ...) {
   if (all(is_color)) {
     colour_ramp(x)
   } else {
-    approxfun(seq(0, 1, length.out = length(x)), x)
+    stats::approxfun(seq(0, 1, length.out = length(x)), x)
```

Comment:
I had ignored this warning because this `as_continuous_pal()` should be deleted in ggplot2 once {scales} releases it. No action needed here, but just wanted to remark upon it.